### ISSUE TITLE
 Unit: frmConConexao.pas

### ISSUE THREAD

**Issue by walwalwalides** [fmbanhos/ERP#1]
Hi nice work !  ,
u get a small access violation error in Unit: frmConConexao.pas
cause u can not free the form, after showing using class procedure methods
u have to extra  free the form instance : on FormDestroy methode
for exemple like this 
procedure TFormConsultaConexao.FormDestroy(Sender: TObject);
begin
  FormConsultaConexao := nil;
  FormConsultaConexao.free;
  inherited;
end;

Good continuation 👍

**Comment by fmbanhos** [comment]
Hi, thanks for the help. You said about the form release by the OnDestroy method. But can I also release using the OnClose method as the example below?
procedure TFormConsultaLancamento.FormClose (Sender: TObject;
  var Action: TCloseAction);
begin
  if not FIsModal then
  begin
    FormConsultaLancamento: = nil;
    Action: = caFree;
  end;
end;

**Comment by walwalwalides** [comment]
yes of course,u can use Onclose mehtode ,it's alMost perform the Same job but more specifique.if it's not the Main Form of your Project ,it better to use on destroy methode.:wink:
